PowerShellTools/DebugEngine/Remote/RemoteEnumDebugProcess.cs

```diff
@@ -98,6 +105,12 @@ public void connect(IDebugPort2 remotePort, bool useSSL)
             }
         }
 
+        private bool isLocalHost()
+        {

```

Comment:
127.0.0.1 is not the only loopback address.  Should use https://msdn.microsoft.com/en-us/library/vstudio/system.net.ipaddress.isloopback(v=vs.100).aspx